curriculum/challenges/english/25-front-end-development/lab-quiz-game/66f17db06803d11a1bd19a20.md

```diff
@@ -19,7 +19,7 @@ Fulfill the user stories below and get all the tests to pass to complete the lab
 1. The `answer` key should have the value of a string, representing the correct answer to the question. Also, the value of `answer` should be included in the `choices` array.
 1. You should have a function named `getRandomQuestion` that takes an array of questions as a parameter and returns a random question object from the array.
 1. You should have a function named `getRandomComputerChoice` that takes the array of the available choices as a parameter, and returns a random answer to the selected question.
-1. You should have a function named `getResults` that takes the selected question object and the computer choice as its parameters and returns `The computer's choice is correct!` if the answer is correct. Otherwise, it returns `The computer's choice is wrong. The correct answer is: <correct-answer>`, where `<correct-answer>` is the value of the correct answer to the chosen question.
+1. You should have a function named `getResults`.It takes 2 parameters: `selectedQuestion` of type object and the `computerChoice` of type string and returns `The computer's choice is correct!` if the answer is correct. Otherwise, it returns `The computer's choice is wrong. The correct answer is: <correct-answer>`, where `<correct-answer>` is the value of the correct answer to the chosen question.
```

Comment:
```suggestion
1. You should have a function named `getResults`. It takes 2 parameters: `selectedQuestion` of type object and the `computerChoice` of type string and returns `The computer's choice is correct!` if the answer is correct. Otherwise, it returns `The computer's choice is wrong. The correct answer is: <correct-answer>`, where `<correct-answer>` is the value of the correct answer to the chosen question.
```